src/Features/LanguageServer/Protocol/Handler/RequestContext.cs

```diff
@@ -66,6 +68,15 @@ namespace Microsoft.CodeAnalysis.LanguageServer.Handler
 
                 if (!documents.IsEmpty)
                 {
+                    if (workspace.IsMiscellaneousFilesWorkspace)
+                    {
+                        Logger.Log(FunctionId.FindDocument_InMiscellaneousFilesWorkspace);
```

Comment:
If you don't pass in a keyvaluelogmessage, it will not get logged to VSTelemetry
http://sourceroslyn.io/#Microsoft.CodeAnalysis.Remote.ServiceHub/VSTelemetryLogger.cs,34